_posts/2018-07-10-react-native-wdio.md

```diff
@@ -0,0 +1,358 @@
+---
+layout: post
+title: "React Native Application UI testing using WebDriverIO and Appium"
+date: 2018-07-09 22:58:15 -0800
+cover: /assets/images/cover-wdio.png
+excerpt: We recently adopted WebDriverIO based UI testing for our React Native application. Benefits of using WebDriverIO include allowing us to write UI tests just as we wrote tests for the web. The [synchronous nature of WebDriverIO](http://webdriver.io/guide/getstarted/v4.html#It%E2%80%99s-all-synchronous) helps us write clean UI tests with PageObject models. WebDriverIO configuration allows us to plugin SauceLabs Emulators or Real Devices for cloud-based testing.
+authors:
+  - name: Raja Panidepu
+    url: https://www.linkedin.com/in/rpanidepu/
+    photo: /assets/images/rpanidepu.jpg
```

Comment:
Looks like the file you are checking in is a PNG. I noticed this rendering in the footer:

<img width="632" alt="screen shot 2018-07-10 at 9 53 49 am" src="https://user-images.githubusercontent.com/1934719/42525177-564263fa-8427-11e8-9f79-310eacaa0cb2.png">